src/dreammaker/builtins.rs

```diff
@@ -434,6 +453,7 @@ pub fn register_builtins(tree: &mut ObjectTree) {
         proc/get_step_rand(Ref);
         proc/get_step_to(Ref,Trg,Min=0);
         proc/get_step_towards(Ref,Trg);
+        proc/gradient(Gradient, index); // unsure how to handle (Item1, Item2, ..., index) form
```

Comment:
curious on if there's a proper way to address the two forms at once

Reply:
no, but it really only affects the popup parameter list completion, so it hasn't been a priority